python_functions.py

```diff
@@ -0,0 +1,32 @@
+x = int(input()) # Ex. 1
+y = int(input())
+
+def division(x, y):
+    c = x/y
+    return c
+
+res = division(x, y)
+print(res)
+
+
+numbers = input() # Ex. 2 (čísla se zadávají oddělená mezerou)
+string_list_of_numbers = numbers.split()
+int_list_of_numbers = []
+for string_number in string_list_of_numbers:
+    string_number_1 = int(string_number)
+    int_list_of_numbers.append(string_number_1)
+
+def sum_of_all_numbers(int_list_of_numbers):
+    start = 0

```

Comment:
`start` je trochu zavadejici, mozna lepsi by bylo `total` ci tak. (`sum` je python klicove slovo pro funkci prave souctu, tak to take radeji ne).